packages/cli/src/Server.ts

```diff
@@ -450,21 +389,16 @@ export class Server extends AbstractServer {
 		// ----------------------------------------
 		this.app.post(
 			`/${this.restEndpoint}/curl-to-json`,
-			ResponseHelper.send(
-				async (
-					req: CurlHelper.ToJson,
-					res: express.Response,
-				): Promise<{ [key: string]: string }> => {
-					const curlCommand = req.body.curlCommand ?? '';
-
-					try {
-						const parameters = toHttpNodeParameters(curlCommand);
-						return ResponseHelper.flattenObject(parameters, 'parameters');
-					} catch (e) {
-						throw new BadRequestError('Invalid cURL command');
-					}
-				},
-			),
+			ResponseHelper.send(async (req: CurlHelper.ToJson): Promise<{ [key: string]: string }> => {
```

Comment:
Inferrable:

```suggestion
			ResponseHelper.send(async (req: CurlHelper.ToJson) => {
```

Also the hand-written signature does match the inferred signature exactly.